templates/live_views/authentication_live.ex

```diff
@@ -165,6 +165,13 @@ defmodule <%= inspect @web_pascal_case %>.AuthenticationLive do
     }
   end
 
+  def handle_info({:error, %{"message" => message, "name" => "NoUserVerifyingPlatformAuthenticatorAvailable"}}, socket) do
+    socket
+    |> assign(:token_form, nil)
+    |> put_toast(:error, message)
```

Comment:
Must be an Irish thing ;)

```suggestion
    |> put_flash(:error, message)
```

Reply:
🤣 This actually made me laugh.